common/app/model/meta.scala

```diff
@@ -173,6 +175,18 @@ object MetaData {
     val sectionSummary: Option[SectionSummary] = apiContent.section map SectionSummary.fromCapiSection
     val sectionId = sectionSummary map (_.id) getOrElse ""
 
+    val atomTypes: Seq[AtomType] = apiContent.atoms.fold[Seq[AtomType]](Nil) { atoms =>
+      Seq(
+        atoms.guides.map(_ => AtomType.Guide),
+        atoms.qandas.map(_ => AtomType.Qanda),
+        atoms.profiles.map(_ => AtomType.Profile),
+        atoms.timelines.map(_ => AtomType.Timeline)
+      ).flatten
+    }
+
+    val atomsCSS: Seq[String] = ArticleAtomRenderer.getCSS(atomTypes)
+    val atomsJS: Seq[String] = ArticleAtomRenderer.getJS(atomTypes)
```

Comment:
I don't think I understand why atom css and js belongs in `metadata`. Can you explain? Thank you

Reply:
Sorry that's me, I did this to make it work even though I knew there would be a better solution eventually. That is where you come in 😅

Reply:
no pb. Ping me when it is ready for review.